docker/msfconsole.rc

```diff
@@ -6,5 +6,5 @@ else
   lhost = %x(hostname -i)
 end
 run_single("setg LHOST #{lhost}")
-run_single("db_connect #{ENV['DATABASE_URL'].gsub('postrgres://', '')}") if ENV['DATABASE_URL']
```

Comment:
At first I had fixed the typo `postrgres` -> `postgres`, but when figured out that `gsub('postrgres://', '')` never actually substitutes anything because it is a typo. You can test it with `db_connect postgres://postgres@db:5432/msf` in msfconsole which works correctly even if used with protocol prefix.